cli/cmds/utils

```diff
@@ -22,7 +22,8 @@ format_address() {
 
 set_rpc_url() {
     case $1 in
-        1) RPC_URL="https://cloudflare-eth.com" ;; # Ethereum
```

Comment:
this RPC continuously has issues and drops transactions, updating to infura

Reply:
Suggest to allow user provide URLs (via env maybe) and default to public one if not provided.